src/app/index.route.coffee

```diff
@@ -160,7 +189,13 @@ angular.module 'mnoEnterpriseAngular'
           controllerAs: 'vm'
 
     $urlRouterProvider.otherwise ($injector, $location) ->
-      unless $injector.get('MnoeConfig').isOnboardingWizardEnabled()
+      MnoeConfig = $injector.get('MnoeConfig')
+      MnoeCurrentUser = $injector.get('MnoeCurrentUser')
+
+      if !MnoeCurrentUser.get() && MnoeConfig.arePublicPagesEnabled()
```

Comment:
`MnoeCurrentUser.get()` returns a promise, I'm not sure if this evaluation is right: `!MnoeCurrentUser.get()`